src/current/_includes/v23.1/backups/storage-collision-examples.md

```diff
@@ -0,0 +1 @@
+For example, if you have a backup schedule running backups for the database `users` the full backup and incremental backup should have the same storage URI for the full and incremental schedule. (`CREATE SCHEDULE FOR BACKUP` will automatically create two schedules for the full and incremental backup to the same storage URI.) If there is another backup schedule, for the database `accounts`, the full and incremental backups for `accounts` should have the same storage URI. However, the storage URI for the `accounts` backup collection, should be different to the storage URI for the `users` database backups.
```

Comment:
same comment here re: the last sentence using the exact same terms for the things being compared